### ISSUE TITLE
Plugin causes .tf-files to become unsearchable

### ISSUE THREAD

**Issue by BeyondEvil** [VladRassokhin/intellij-hcl#352]
Thank you for opening an issue.
In this template paragraph text could be removed, however please retain headers.

### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates

### Installation details
 - [x] IDE version (`Help->About->Copy to Clipboard`)
 ```
IntelliJ IDEA 2021.1 (Ultimate Edition)
Build #IU-211.6693.111, built on April 6, 2021
Licensed to [redacted]
Subscription is active until September 11, 2021.
Runtime version: 11.0.10+9-b1341.35 x86_64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.7
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8

Kotlin: 211-1.4.32-release-IJ6693.72
```
 - [x] intellij-hcl plugin version (`Settings->Plugins`)
 `0.7.10`
 - [x] Terraform version (`terraform -v`)
 `0.12.31`

### Terraform Configuration Files
```hcl
n/a
```

### Exception
```text
n/a
```

### Expected Behavior
What should have happened?
Searching for strings using `Find in Files...` shows "no occurences" even tho string exists in .tf-file.

### Actual Behavior
What actually happened?
Search should produce occurences in .tf-files.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
Search for a string using "Find in Files..." that exists in a .tf-file.

**Comment by VladRassokhin** [comment]
Cannot reproduce. Probably fixes in 0.7.11

**Comment by VladRassokhin** [comment]
Please reopen if still actual
